app/models/concerns/with_description.rb

```diff
@@ -2,11 +2,8 @@ module WithDescription
   extend ActiveSupport::Concern
 
   included do
-    markdown_on :description, :description_teaser, skip_sanitization: true
+    markdown_on :description, skip_sanitization: true
```

Comment:
We could get rid of this one, right?

Reply:
Do you mean, the  `description` or the `teaser`?

Reply:
I did a quick search and it looks like they are being used in laboratory.. I didn't perform a deep search, though

Reply:
Oh my bad, I thought it said description_teaser which is being defined in teaser_on. Never mind me :sweat: